lib/config-schema.coffee

```diff
@@ -39,6 +39,23 @@ module.exports =
         type: "boolean"
         default: true
 
+      audioclip:
+        title: "Play Audio - Audioclip"
+        description: "Which audio clip played at keystroke."
+        type: "string"
+        default: '../audioclips/gun.wav'
+        enum: [
+          {value: '../audioclips/gun.wav', description: 'Gun'}
+          {value: '../audioclips/typewriter-key-1.wav', description: 'Type Writer'}
```

Comment:
I hate to be so picky, but I don't want to add another audio, I think the custom audio option is enough for now.